checkov/terraform/graph_builder/local_graph.py

```diff
@@ -603,6 +603,11 @@ def update_nested_modules_address(self) -> None:
             context_parser = parser_registry.context_parsers[vertex.block_type]
             entity_context_path = context_parser.get_entity_context_path(vertex.config)
             resource_id = '.'.join(entity_context_path) if entity_context_path else vertex.name
+
+            if vertex.block_type == BlockType.MODULE:
+                # prefix modules with 'module' to properly identify them
+                resource_id = f"{BlockType.MODULE}.{resource_id}"
```

Comment:
how come we don't need to do it for any other resource type?

Reply:
currently none of the other non-resource blocks has a prefix, which is not so important, because they are identifiable by their block type. I did this change to align the address field to be identical to the prefix for other block types inside a module.